Wangscape/noise/module/Reseedable.cpp

```diff
@@ -0,0 +1,39 @@
+#include "Reseedable.h"
+
+namespace noise
+{
+namespace module
+{
+
+template<> void Reseedable<ModuleGroup>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Reseedable<Billow>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Reseedable<Perlin>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Reseedable<RidgedMulti>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Reseedable<Turbulence>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Reseedable<Voronoi>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
```

Comment:
The libnoise classes with `SetSeed` don't implement it as a virtual function of the module base class (because there are modules that don't need a seed), so these are all "different" functions. At the moment, this is the simplest way to gather them all.